LocalSettings.php

```diff
@@ -533,8 +533,6 @@
 			"$IP/extensions/CheckUser/cu_changes.sql",
 			"$IP/extensions/DataDump/sql/data_dump.sql",
 			"$IP/extensions/Echo/echo.sql",
-			"$IP/extensions/GlobalBlocking/sql/global_block_whitelist.sql",
```

Comment:
this is needed locally, it allows locally disabling certain global blocks

Reply:
Reverted in https://github.com/miraheze/mw-config/commit/85ea53fac46b2e381e9dea7a81c647ab62fdc965